### ISSUE TITLE
External js files (import by main entry file) built without obfuscation or minify

### ISSUE THREAD

**Issue by jooy2** [electron-vite/vite-plugin-electron#86]
My project has the following directory structure:

```text
src
    ㄴ main
        ㄴ utils
            test.js
        index.js
    ㄴ renderer
        main.js
        ...
```

If i build the package using `vite-plugin-electron`, the build succeeds, but when i import and use an external js file like the `src/main/utils/test.js` file, the file is built without minify or code obfuscation. The `main/utils/test.js` file in the build output path is the same as the existing file.

Is there a way to obfuscate (or minify) external js files other than the main entry file? Or is this a bug?

here is my `vite.config.js`:

```javascript
import { defineConfig } from 'vite'
import electron from 'vite-plugin-electron'
import vue from '@vitejs/plugin-vue'
import { resolve, join } from 'path'
import { builtinModules } from 'module'

export default defineConfig({
  base: './',
  root: resolve('./src/renderer'),
  publicDir: resolve('./src/renderer/public'),
  build: {
    outDir: resolve('./dist')
  },
  plugins: [
    vue(),
    electron({
      main: {
        entry: 'src/main/index.js',
        vite: {
          publicDir: resolve('./src/main'),
          build: {
            emptyOutDir: true,
            assetsDir: '.',
            outDir: 'dist/main',
            rollupOptions: {
              external: [
                'electron',
                ...builtinModules
              ]
            }
          }
        }
      },
      renderer: {}
    })
  ]
})
```

Thank you.

**Comment by jooy2** [comment]
Thanks for the quick reply.

I try to write efficient code by splitting multiple actions into each file.

So is there any other way to minify multiple files? I'd like to know how, if possible.
If not, are there any plans to fix it in the future?

Thanks,

**Comment by jooy2** [comment]
Thanks for the clear explanation! That is a good news. I will wait until it is officially supported, and i will check it as soon as it is supported.

You can close this issue now or after supported.

**Comment by caoxiemeihao** [comment]
Hi here! 👋
The feature implemented in `v0.10.0` #89
